### DISCUSSION TITLE
Ошибка 404 при выполнении дефолтного скрипта

### DISCUSSION THREAD

**Discussion by o000oo799** [Chimildic/goofy#202] [Помощь с ошибкой] [answered]
Выбивает ошибку 404. Поскольку только зарегистрировался, беру пока только Release Radar, без открытий недели
ссылка на плейлист https://open.spotify.com/playlist/37i9dQZEVXbmdqiZQSxTCV?si=76e43ba364f94211

function createFirstPlaylist() {
 // 1 - собираем
 let mixTracks = Source.getTracks([
     { name: 'Микс дня 1', id: '37i9dQZEVXbmdqiZQSxTCV' },
     //{ name: 'Микс дня 2', id: 'вашеId' },
 ]);
 let recomTracks = Source.craftTracks(mixTracks);
 let savedTracks = Source.getSavedTracks();   

 // 2 - обрабатываем
 Combiner.push(mixTracks, recomTracks);   
 Filter.removeTracks(mixTracks, savedTracks);
 Selector.keepRandom(mixTracks, 20);
 
 // 3 - создаем плейлист
 Playlist.saveWithReplace({
      id: '3780273512f04587be1325ce0a3f9298',
     name: 'Первый плейлист333',
     tracks: mixTracks,
     randomCover: 'update',
 });
}

**Comment by o000oo799** [comment]
Все заработало

**Comment by Chimildic** [comment, marked as answer]
id целевого плейлиста `3780273512f04587be1325ce0a3f9298` ошибочно, слишком много символов и в целом не стандартный паттерн

**Reply by o000oo799** [reply]
так туда же не id целевого плейлиста, а id юзера пишется
вот исходный скрипт из goofy 
function createFirstPlaylist() {
 // 1 - собираем
 let mixTracks = Source.getTracks([
     { name: 'Микс дня 1', id: 'вашеId' },
     { name: 'Микс дня 2', id: 'вашеId' },
 ]);
 let recomTracks = Source.craftTracks(mixTracks);
 let savedTracks = Source.getSavedTracks();   

 // 2 - обрабатываем
 Combiner.push(mixTracks, recomTracks);   
 Filter.removeTracks(mixTracks, savedTracks);
 Selector.keepRandom(mixTracks, 20);
 
 // 3 - создаем плейлист
 Playlist.saveWithReplace({
     // id: 'вашеId',
     name: 'Первый плейлист',
     tracks: mixTracks,
     randomCover: 'update',
 });
}

**Reply by Chimildic** [reply]
Нет, туда пишется id плейлиста, в который нужно сохранить треки. 
Подробнее в пункте №6 из [гайда о первом плейлисте](https://chimildic.github.io/goofy/#/first-playlist).
